Makefile

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Are we keeping this initial blank line around for some reason?  If not let's drop.

Reply:
Removed.